-------------------------------------------------------------------------------
Test set: com.ruijie.redis.RedisTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 11.962 s <<< FAILURE! - in com.ruijie.redis.RedisTest
test(com.ruijie.redis.RedisTest)  Time elapsed: 2.518 s  <<< ERROR!
org.springframework.data.redis.RedisConnectionFailureException: Unable to connect to Redis; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to localhost:6379
	at com.ruijie.redis.RedisTest.test(RedisTest.java:24)
Caused by: io.lettuce.core.RedisConnectionException: Unable to connect to localhost:6379
	at com.ruijie.redis.RedisTest.test(RedisTest.java:24)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
Caused by: java.net.ConnectException: Connection refused: no further information

